lib/Conch/Controller/ValidationPlan.pm

```diff
@@ -49,8 +49,7 @@ sub create ($c) {
 
 	$c->log->debug("Created validation plan ".$validation_plan->id);
 
-	# FIXME why is this not using the same redirect methodology as the rest of the api?
-	$c->status( 201, $validation_plan );
+	$c->status(303 => "/validation_plan/".$validation_plan->id);
```

Comment:
201 is more correct, I think, so I'd be happy to arrange a time when we consistently change to the correct way and change -cli and -ui at the same time to match.

Reply:
Agreed. But with one exception, the whole API uses 303 and I'm opting for consistency over correctness at the moment.